compose-builder/add-security.yml

```diff
@@ -75,13 +75,13 @@ services:
     environment:
       # uncomment and modify the following line to add additional registry ACL roles on the fly
       # the list is comma-separated service keys for these services
-      #ADD_REGISTRY_ACL_ROLES: edgex-app-service-configurable-rules, device-virtual
+      #ADD_REGISTRY_ACL_ROLES: appservice-sample,appservice-rules-engine-redis, appservice-rules-engine-mqtt, appservice-push-to-core
```

Comment:
change to `app-sample, app-rules-engine-redis, app-rules-engine-mqtt, app-push-to-core`

Reply:
fixed